lib/games_manager.rb

```diff
@@ -67,6 +67,14 @@ def average_goals_per_game
     ratio(total_goals, total_games)
   end
 
+  def average_goals_by_season
+    avg_goals_by_season = {}
+    seasonal_game_data.each do |season, details|
+      avg_goals_by_season[season] = ratio(total_goals(details), total_games(details))
+    end
+    avg_goals_by_season
+  end
```

Comment:
Maybe this could be refactored using a reduce - Hope or Jake would probably be better at doing this - just to save a couple lines of code.

Reply:
Yeah, it sure looks like it could be a reduce. I might just agree to the PR and fix it on my end.